.ci/windows-steps.yaml

```diff
@@ -12,9 +12,12 @@ steps:
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_random-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install unofficial-flayan-cereal -o $(Agent.ToolsDirectory)
-
+    nuget install ensmallen -o $(Agent.ToolsDirectory) -Version 2.17.0
+    dir $(Agent.ToolsDirectory)
+    tree $(Agent.ToolsDirectory)
+    ## Delete all ensmallen dependencies, we do not need them here
+    Remove-Item $(Agent.ToolsDirectory)\ensmallen.2.17.0\installed\x64-linux\share -Force -Recurse
```

Comment:
@rcurtin I have already done that, I suppose that maybe it is not working? I will try to verify if the directory is still there

Reply:
Ah, sorry, I missed that.  Well, maybe it is worth printing everything in the ensmallen directory, just to make sure the headers are not there.

Reply:
What I find strange is that it has never been able to link with any armadillo version including the one that is in ensmallen package. However, when I corrected the path to add the `tree\include`, it showed behaviour similar as if it was linking with the package.
I have added a tree to see if armadillo is not deleted from the ensmallen package, let us see.

Reply:
@rcurtin we are sure now that visual studio is trying to use the local headers, which is causing all of these issues.
Have a look here
https://dev.azure.com/mlpack/mlpack/_build/results?buildId=7292&view=logs&j=934eac7e-378e-5e05-ed87-f05e34fa9fdc&t=4ee4870b-c6ca-5a83-cdbe-639dd57a3582&l=9973

Reply:
Take a look at the error:

```
##[error]C:\hostedtoolcache\windows\ensmallen.2.17.0\installed\x64-linux\include\armadillo(21,10): Error C1083: Cannot open include file: 'armadillo_bits/compiler_check.hpp': No such file or directory
```

I think that you have removed everything of Armadillo that's distributed with ensmallen... *except* the `armadillo` file. So if you remove that too, in addition to the `armadillo_bits/` directory, then I think we will be closer to something that works. :)